cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java

```diff
@@ -298,12 +299,15 @@ public boolean accept(String className, URL classUrl, URL classPathUrl) {
     for (File dep : orderedDependencies) {
       builder.add("file://" + dep.getAbsolutePath());
     }
-    List<String> orderedDependenciesStr = builder.build();
+    List<String> orderedDependenciesStr = Lists.newArrayList(builder.build());
 
     System.setProperty(HiveConf.ConfVars.HIVEAUXJARS.toString(), Joiner.on(',').join(orderedDependenciesStr));
     LOG.debug("Setting {} to {}", HiveConf.ConfVars.HIVEAUXJARS.toString(),
               System.getProperty(HiveConf.ConfVars.HIVEAUXJARS.toString()));
 
+    // add hive-exec.jar to the end of auxjars, we need this, otherwise when hive runs a MapRedLocalTask it cannot find
+    // "org.apache.hadoop.hive.serde2.SerDe" class in its classpath.
+    orderedDependenciesStr.add(new HiveConf().getJar());
```

Comment:
Since `hive-exec.jar` is only required for local mapreduce task, it would be better to not add it to `orderedDependenciesStr`, but add it only while giving the dependencies to `LocalMapreduceClasspathSetter`